style/servo/media_queries.rs

```diff
@@ -301,6 +301,11 @@ impl Device {
         AbsoluteColor::BLACK
     }
 
+    /// Returns color scheme of the current device
+    pub fn color_scheme(&self) -> PrefersColorScheme {
+        self.prefers_color_scheme
+    }
+
```

Comment:
Seems fine. I have a bit of a nit with the documentation:

```suggestion
    /// Returns the color scheme of this [`Device`].
    pub fn color_scheme(&self) -> PrefersColorScheme {
        self.prefers_color_scheme
    }

```

Reply:
oh thanks, i'm not versed in doc comments